src/assets/templates/widgets/sir/sir_item.html

```diff
@@ -138,9 +138,9 @@ <h3>Next Steps</h3>
           </div>
           <div class="cc-widget-sir-button-container">
             <button
-              class="cc-button cc-button-blue"
-              type="submit"
-              data-ng-disabled="!api.user.profile.actAsOptions.canPost || !sirItem.isFormValid || sirItem.isSubmitting"
+              ng-class="setButtonClass()"
+              data-ng-disabled="!api.user.profile.actAsOptions.canPost || !sirItem.form.option || sirItem.isSubmitting"
```

Comment:
@frankiepun - I wonder if this logic can be shared in the controller in a function called `formDisabled()` and used with `$scope.setButtonClass` also... or is the logic much different between the two?

Reply:
Thanks Jason. I always forgot to add the PR to the ticket. I have just added it.

Excellent point of sharing the logic! The user has a special requirement of two button-states: a normal disabled, non-clickable button and a clickable greyed-out button. data-ng-disabled handles the normal disabled button case. The clickable button has one more condition in setButtonClass(). Let me try to consolidate the logic.